static/styles/subscriptions.css

```diff
@@ -603,21 +603,6 @@ h4.stream_setting_subsection_title {
     margin-bottom: 10px;
 }
 
-.stream-creation-body {
-    section.block {
-        margin-bottom: 20px;
-    }
-
-    #announce-new-stream {
-        margin-top: 10px;
-
-        div[class^="fa"] {
-            margin-left: 3px;
-            margin-right: 8px;
-        }
-    }
-}
-
```

Comment:
Instead of again using the classname `.stream-creation-body` to write CSS for `section.block` keep them nested inside where it's already defined. 

For reference ->